### DISCUSSION TITLE
Non-US accounts not working?

### DISCUSSION THREAD

**Discussion by kresov** [nVuln/homebridge-lg-thinq#123] [General]
Hi, I'm having trouble using the plugin with non-US account.

Message displayed is:
Login error. Need Sign-In Unlock:5:********@gmail.com:Y

With 3 letters of my email address, I've just removed them here.
Account country is Croatia.

Tested with US account and it logs in without problem, but to use it I'd have to change service country for my devices etc. I'd rather use correct one :)

**Comment by nVuln** [comment]
please try login your account in LG ThinQ mobile app first
did you changed `country` in plugin config ?

**Reply by kresov** [reply]
Yep, I have 2 devices (TV and Washer/Dryer) added in mobile app.

I didn't manually edit the JSON, but I've set correct country in UI dropdown.
Tried Croatia/Croatian and Croatia/English too.

Here's a test from CLI:

root@raspberrypi:/homebridge# thinq -c HR -l EN login myemail mypassword
Start login: username = myemail , password = mypassword , country = HR , language = EN
AuthenticationError: Need Sign-In Unlock:5:kvi*****@gmail.com:Y
    at /homebridge/node_modules/homebridge-lg-thinq/dist/lib/Auth.js:85:19
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async Auth.loginStep2 (/homebridge/node_modules/homebridge-lg-thinq/dist/lib/Auth.js:77:25)
    at async Command.<anonymous> (/homebridge/node_modules/homebridge-lg-thinq/dist/cli.js:54:25)

Thanks for the quick reply BTW

**Reply by nVuln** [reply]
please try signout and re-signin on mobile app
and double check if your account is verified email

**Reply by kresov** [reply]
That worked. Huh.

Thanks!
